app/queries/get_activity_log_events.rb

```diff
@@ -0,0 +1,26 @@
+class GetActivityLogEvents
+  attr_reader :application_choices
+
+  def initialize(application_choices:)
+    @application_choices = application_choices
+  end
+
+  def call(since: nil)
+    since ||= Time.zone.local(2018, 1, 1)
```

Comment:
can this time be moved into a constant. What does it signify? Is this when the project begun?

Reply:
It doesn't really mean anything... It's just a date before the pilot started so that the service returns all relevant records, effectively not limiting results based on time. I didn't want to add any conditionals in the code below. Suggestions for alternatives welcome.